### PR TITLE
"characters" endpoint implementation

### PR DESCRIPTION
**Pull request by fanonwue**
I've started working on implementing the "characters" endpoint. I'm opening this pull request to let other people know ;)

The tests are still very much WIP as I have never worked with this test framework.

### CODE REVIEW

**Comment by Kryszak** [comment]
I'm actually glad that you picked up the missign features, I don't have that much time as I had when I started this project to work on private stuff.
As for tests, it should be fairly simple - all of them are pretty much the same :)

File: src/test/groovy/com/kryszak/gwatlin/api/characters/CharactersClientSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+import com.kryszak.gwatlin.api.characters.model.character.sab.SabZoneMode
+import com.kryszak.gwatlin.api.shared.ItemBinding
+import com.kryszak.gwatlin.api.shared.ItemSlot
+import com.kryszak.gwatlin.config.WiremockConfig
+import spock.lang.Subject
+
+
+class CharactersClientSpec extends WiremockConfig {
+
+    final TARGET_SCHEMA_VERSION = "2021-07-15T13:00:00.000Z"
```
Comment (Kryszak) [inline src/test/groovy/com/kryszak/gwatlin/api/characters/CharactersClientSpec.groovy:32]: Please use either `final` or `def` keywords for both fields for consistency

**Comment by Kryszak** [comment]
One minor cosmetic change, apart from that all looks good.
I resolved Codacy issues, so after that change in test PR will be good to go

**Comment by fanonwue** [comment]
Yeah should have stayed consistent there. Fixed it now.

**Comment by Kryszak** [comment]
Great, I'll merge this and release new version later this day.

**Pull request merged**